queries/elixir/aerial.scm

```diff
@@ -24,6 +24,15 @@
   (#set! "kind" "Function")
   ) @type
 
+(call
+  target: (identifier) @identifier (#any-of? @identifier "def" "defp" "defguard" "defmacro" "defmacrop")
+  (arguments [
+              ((identifier) @name)
+   ])
+  (#set! "kind" "Function")
+  ) @type
+
+
```

Comment:
Is it possible to roll this into the above query by adding the `((identifier) @name)` to the alternation (`[]`)?

Reply:
indeed you're right. had no idea [] is alternation in treesitter's syntax... this is what happens when one "learns by doing" instead of reading the docs :)